suites/quincy/cephadm/tier-1_mix_os.yaml

```diff
@@ -0,0 +1,158 @@
+#=========================================================================================================
```

Comment:
rename to `tier-1-mix-os-deployment.yaml`

Reply:
Updated